pages/dkp/kommander/1.3/clusters/index.md

```diff
@@ -1,53 +1,53 @@
 ---
 layout: layout.pug
-beta: false
+beta: true
 navigationTitle: Managing Clusters
 title: Managing Clusters
 menuWeight: 7
 excerpt: View clusters created with Kommander or any connected Kubernetes cluster
 ---
 
-## Creating or Connecting Clusters
+## Create or connect a clusters
 
-Kommander allows you to monitor and manage very large numbers of clusters and so we make it easy to either connect existing clusters or create new clusters whose lifecycle is managed by Konvoy.
+Kommander allows you to monitor and manage very large numbers of clusters, so it's easy to either connect existing clusters or create new clusters whose lifecycle is managed by Konvoy.
 
 ## Before you begin:
 
-You must have run `konvoy up` with the Kommander addon deployed. You should be looking at the Kommander UI.
-If you are creating a cluster with a cloud partner (Azure, AWS), you must configure your [infrastructure provider](/dkp/kommander/1.3/operations/infrastructure-providers/).
-If you are planning on attaching more than one cluster, ensure you enter your valid [license](/dkp/kommander/1.3/licensing/).
+- You must have run `konvoy up` with the Kommander addon deployed. You should be looking at the Kommander UI.
+- If you are creating a cluster with a cloud partner (Azure, AWS), you must configure your [infrastructure provider](/dkp/kommander/1.3/operations/infrastructure-providers/).
+- If you are planning on attaching more than one cluster, ensure you enter your valid [license](/dkp/kommander/1.3/licensing/).
 
-From the dashboard page or the clusters page, click Add Cluster and Create Konvoy Cluster.
+From the dashboard page or the clusters page, select **Add Cluster > Create Konvoy Cluster**.
 
 ![Add Cluster Options](/dkp/kommander/1.3/img/add-cluster.png)
 
-### Creating a Konvoy Cluster
+### Create a Konvoy cluster
 
 - [Creating Konvoy Clusters on AWS](/dkp/kommander/1.3/clusters/creating-konvoy-cluster-aws/)
 - [Creating Konvoy Clusters on Azure](/dkp/kommander/1.3/clusters/creating-konvoy-cluster-azure/)
 - [Creating Konvoy Clusters on your own infrastructure](/dkp/kommander/1.3/clusters/creating-konvoy-cluster-on-prem/)
 
-### Connecting an Existing Cluster
+### Connect an existing cluster
 
 Using the Add Cluster option, you can attach an already existing cluster, even an existing Konvoy cluster, directly to Kommander. Enjoy all the multi-cluster management and monitoring benefits that Kommander provides while keeping your existing cluster on its current provider & infrastructure.
```

Comment:
I realize this text wasn't changed in this PR, but isn't this the 'Attach Cluster' option?

```suggestion
Using the Attach Cluster option, you can attach an already existing cluster, even an existing Konvoy cluster, directly to Kommander. Enjoy all the multi-cluster management and monitoring benefits that Kommander provides while keeping your existing cluster on its current provider & infrastructure.
```